bridge-impl/src/main/java/com/liferay/faces/bridge/internal/BridgeURIImpl.java

```diff
@@ -431,6 +413,45 @@ public String toString() {
 		return stringValue;
 	}
 
+	private String getParameter(String name, boolean remove) {
+
+		Map<String, String[]> parameterMap = getParameterMap();
+		String[] values;
+
+		if (remove) {
+
+			values = parameterMap.remove(name);
+			invalidateToString();
+		}
+		else {
+			values = parameterMap.get(name);
+		}
+
+		if ((values == null) || (values.length == 0)) {
+
+			String namespace = FacesContextHelperUtil.getNamespace();
```

Comment:
This fixes the problem where the (namespaced) `_jsfBridgeRedirect=true` parameter appeared after a non-Ajax redirect. Since the parameter map was *not* checked for namespaced parameters, the (namespaced) `_jsfBridgeRedirect=true` parameter was not removed.